src/utilities/authentication.js

```diff
@@ -0,0 +1,71 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+/**
+ * scrambles string data
+ * @param {*} a - input string data
+ * @returns {output} - scrambled data
+ */
+function scramble(a) {
```

Comment:
Make the variable name more descriptive. Eg.

`const reverseToken = (token) => token.split('').reverse().join('');`

The above is basically what I'm asking for